### PR TITLE
Add Vagrant configuration

### PR DESCRIPTION
**Pull request by mnapoli**
Tested to be working.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Vagrant.configure(2) do |config|
+
+    config.vm.box = "ubuntu/trusty64"
+
+    config.vm.provision :shell, :path => "vagrant/bootstrap.sh"
+
+    config.vm.provider "virtualbox" do |v|
+        v.customize ["modifyvm", :id, "--memory", 1024]
+    end
+
```
Comment (K-Phoen) [inline Vagrantfile:10]: Could we add a synced folder to export rulerz into the VM? (even if commented by default)

Something like this:

```
Vagrant.configure(2) do |config|
    # …

    # uncomment these lines to share rulerz's repository with the VM
    #config.vm.network "private_network", type: "dhcp"                                                                                                                                                            
    #config.vm.synced_folder ".", "/home/vagrant/workspace", type: "nfs"
end
```

**Comment by K-Phoen** [comment]
What about installing elasticsearch in the VM? It could be useful for contributors.

**Comment by mnapoli** [comment]
@K-Phoen it's not used by tests as there's no functional test for elasticsearch. I've added it in that branch though: https://github.com/wizacha/rulerz/commits/results

File: Vagrantfile (hunk 1 above)
Comment (mnapoli) [inline Vagrantfile:10]: In vagrant the local directory is automatically mounted as `/vagrant`, is that what you meant?

Comment (K-Phoen) [inline Vagrantfile:10]: Yep, I meant that. I haven't used Vagrant in a long time so I didn't know that the local directory was already shared.

Thanks :)

**Comment by K-Phoen** [comment]
The tests will need it soon: https://github.com/K-Phoen/rulerz/tree/behat

I'm using behat in addition to phpspec to provide another style of tests.

**Comment by mnapoli** [comment]
Pushed with Elasticsearch.

**Comment by K-Phoen** [comment]
Perfect, thanks :+1:

**Comment by KuiKui** [comment]
@mnapoli t'es sérieux là ?

**Comment by mnapoli** [comment]
Haha les ptits trolls ! Vagrant ça a du sens ici parce qu'il faut des BDD et tout :p

**Pull request merged**